tests/test_font.py

```diff
@@ -46,6 +46,37 @@ def test_FontArguments_setCollectionIndex(fontarguments):
     fontarguments.setCollectionIndex(0)
 
 
+def test_FontArguments_setPalette(fontarguments):
```

Comment:
You need to add test here to guard against the segfault I mentioned, and also protect against the segfault in the actual code.

Reply:
I already check for passing empty `skia.FontArguments.Palette.Overrides()` and it does not seem to crash.

Reply:
The crash is intermittent, depending on memory layout